pulsar-transformations/src/test/java/com/datastax/oss/pulsar/functions/transforms/Utils.java

```diff
@@ -57,6 +59,15 @@ public static TestTypedMessageBuilder<?> process(Record<GenericObject> record, T
     return context.getOutputMessage();
   }
 
+  public static Record<Object> process2(Record<GenericObject> record, TransformStep step)
```

Comment:
nit: Just use "process"

Reply:
Oh yes, forgot to cleanup. Thanks for noticing.